src/uct/ib/base/ib_md.c

```diff
@@ -1859,7 +1919,7 @@ static void uct_ib_md_vfs_init(uct_md_h md)
 
 static uct_ib_md_ops_t uct_ib_verbs_md_ops = {
     .open                = uct_ib_verbs_md_open,
-    .cleanup             = (uct_ib_md_cleanup_func_t)ucs_empty_function,
+    .cleanup             = uct_ib_md_cleanup,
```

Comment:
maybe we can remove "cleanup" from uct_ib_md_ops_t altogether?
and uct_ib_md_close will do what uct_ib_md_cleanup is doing

Reply:
but it couldn't replace `uct_ib_mlx5_devx_md_cleanup` which does additional work